scripts/gwas/gwas.py

```diff
@@ -488,7 +488,8 @@ def make_saige_commands(args, logistic, step):
 def pass_arguments_along(args, args_list):
     opts = vars(args)
     vals = [opts[arg.replace('-', '_')] for arg in args_list]
-    return ''.join([(' --{} {} '.format(arg, '' if val else val) if val else '') for arg, val in zip(args_list, vals)])
+    l = [(' --{} {} '.format(arg, '' if (val is True) else val) if val else '') for arg, val in zip(args_list, vals)]
```

Comment:
naming this `args` or similar should make flake8 happy (doesn't like single-character object names).